src/test/java/uk/gov/pay/adminusers/infra/SqsTestDocker.java

```diff
@@ -31,19 +31,13 @@ private static void createContainer() {
         if (sqsContainer == null) {
             logger.info("Creating SQS Container");
 
-            sqsContainer = new GenericContainer("mvisonneau/alpine-sqs:1.2.0")
+            sqsContainer = new GenericContainer("softwaremill/elasticmq-native")
```

Comment:
Is this container change related to M1 support? If it is, it'd be good to write an explanation in the commit message.

Reply:
hopefully the commit message if more clarifying now.